string status
float64 dock_status
float64 plat_vel
float64 quad_vel
float64 rel_vel